src/sentry/api/endpoints/organization_incident_index.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+from sentry import features
+from sentry.api.bases.organization import (
+    OrganizationEndpoint,
+    OrganizationPermission,
+)
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.incidents.models import Incident
+
+
+class OrganizationIncidentPermission(OrganizationPermission):
+    scope_map = {
+        'GET': ['org:read', 'org:write', 'org:admin'],
+        'POST': ['org:read', 'org:write', 'org:admin'],
```

Comment:
Should people with only read access to the org be able to create incidents?

Reply:
Oops, left this in from debugging CI